docs/developers/celestia-app-vesting.md

```diff
@@ -382,8 +382,8 @@ celestia-appd tx vesting create-vesting-account $TO_ADDRESS 100000utia 168674805
 Running a consensus full node or validator will prevent you from needing
 to use an RPC.
 
-You can [set up a validator](../../nodes/full-consensus-node) or [full node](../../nodes/validator-node) for the previous portion
-of the tutorial.
+You can [set up a validator or full node](../../nodes/consensus-node)
```

Comment:
[optional] the previous sentence clarifies that the full node is a consensus full node so proposal to repeat

```suggestion
You can [set up a validator or consensus full node](../../nodes/consensus-node)
```

Reply:
matched this PR! https://github.com/celestiaorg/docs/commit/96a13977fe56900518d6423215a041ee44930b71 good flag